content/guides/astro.md

```diff
@@ -17,11 +17,172 @@ comments: false
 draft: false
 ---
 
+{{< hextra/hero-subtitle >}}
+  Astro is an all-in-one web framework for building content-driven websites, that pioneers a new frontend architecture to reduce JavaScript overhead and complexity. Learn in this guide how to deploy an Astro site on Clever Cloud.
+{{< /hextra/hero-subtitle >}}
+
+## Overview
+
+Clever Cloud supportes deploying both fully static and on-demand rendered Astro projects. Regardless of your `output` mode ([pre-rendered or on-demand](/en/basics/rendering-modes/)), you can choose to deploy as a **static application** which runs using a post-build hook, or as a **Node.js** application, which works out-of-the-box with your `package.json`.
+
 If you need an example source code, get [Astrowind](https://github.com/onwidget/astrowind) (you'll need [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and [Node.js](https://nodejs.org/en/learn/getting-started/how-to-install-nodejs)):
 ```bash
 git clone https://github.com/onwidget/astrowind myStaticApp
 ```
 
+## Port and host
+
+Applications on Clever Cloud listen on port **8080**. If your project requires this configuration, set your port and host in Astro in one of two locations:
+
+{{< tabs items="package.json scripts,astro.config.mjs" >}}
+
+  {{< tab >}}
+
+   ```json {filename="package.json"}
+    "scripts": {
+      "dev": "astro dev",
+      "start": "astro dev",
+      "build": "astro check && astro build",
+      "preview": "astro preview --host 0.0.0.0 --port 8080",
+      "astro": "astro"
+    } 
+    ```
```

Comment:
Some parts of the file are not needed, start for example as we use a `CC_RUN_COMMAND`

Reply:
It can be set up both ways.